GliaWidgets/ViewController/GliaViewController.swift

```diff
@@ -20,10 +20,6 @@ class GliaViewController: UIViewController {
     private var sceneProvider: SceneProvider?
```

Comment:
MUIC-358 Bring back custom transition animations for Glia views + MUIC-508 that refactors this and fixes some issues